### PR TITLE
Adds dotnet codec server sample

### PR DESCRIPTION
**Pull request by rachfop**

### CODE REVIEW

File: docs-src/concepts/what-is-a-codec-server.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Most SDKs provide example Codec Server implementation samples, listed here:
 
 - [Go](https://github.com/temporalio/samples-go/tree/main/codec-server)
 - [Java](https://github.com/temporalio/sdk-java/tree/master/temporal-remote-data-encoder)
+- [.Net](https://github.com/temporalio/samples-dotnet/tree/main/src/Encryption)
```
Comment (axfelix) [inline docs-src/concepts/what-is-a-codec-server.md:27]: I think this is supposed to be ".NET" generally but definitely not worth blocking a small addition until we have an SDK

**Review by axfelix — APPROVED** [review]

File: docs-src/concepts/what-is-a-codec-server.md (hunk 1 above)
Comment (rachfop) [inline docs-src/concepts/what-is-a-codec-server.md:27]:
```suggestion
- [.NET](https://github.com/temporalio/samples-dotnet/tree/main/src/Encryption)
```

File: docs/concepts/dataconversion.md [hunk 2]
```diff
@@ -257,6 +257,7 @@ Most SDKs provide example Codec Server implementation samples, listed here:
 
 - [Go](https://github.com/temporalio/samples-go/tree/main/codec-server)
 - [Java](https://github.com/temporalio/sdk-java/tree/master/temporal-remote-data-encoder)
+- [.Net](https://github.com/temporalio/samples-dotnet/tree/main/src/Encryption)
```
Comment (rachfop) [inline docs/concepts/dataconversion.md:260]:
```suggestion
- [.NET](https://github.com/temporalio/samples-dotnet/tree/main/src/Encryption)
```

File: docs-src/concepts/what-is-a-codec-server.md (hunk 1 above)
Comment (rachfop) [reply docs-src/concepts/what-is-a-codec-server.md]: Thanks, fixed!

**Pull request merged**